io/pci/dlpar.py

```diff
@@ -57,13 +57,7 @@ def setUp(self):
                                password=self.hmc_pwd)
         if not self.session.connect():
             self.cancel("failed connecting to HMC")
-        cmd = 'lssyscfg -r sys  -F name'
-        output = self.session.cmd(cmd)
-        self.server = ''
-        for line in output.stdout_text.splitlines():
-            if line in self.lpar_1:
-                self.server = line
-                break
+        self.server = self.params.get("manageSystem", '*', default=None)
```

Comment:
why does `path` need to be provided I did not feel it is relevant here or maybe I am not able to make it out, can you please help me to understand in a better way

Reply:
ah copy paste error,  even other param has the it .. removed
self.hmc_pwd
self.lpar_2
self.pci_device